src/lib/validateAddress.js

```diff
@@ -1,4 +1,4 @@
-import { isAddress as isEthAddress, isHexStrict } from 'web3-utils'
+import { isAddress as isEthAddress, isHexStrict } from 'web3-validator'
```

Comment:
Can you convert this file to TS and write some unit tests for `validateAddress`?